inginious/frontend/plugins/contact_page/static/js/contact_page_form.js

```diff
@@ -76,6 +95,31 @@ class ContactPageForm {
         const self = this;
         this.sendButton.click(function () {
             self.sendInfo();
+            self.sendButton.prop("disabled",true);
+        });
+    }
+
+    addCloseListenerToOnCloseModal() {
+        const self = this;
+        this.modalSendMessage.on("hidden.bs.modal", function() {
+            if (self.alertOnModal.is('.alert-success')) {
+                if (self.editCheckbox.is(":checked")) {
+                    self.emailInput.val('');
+                    self.nameInput.val('');
+                }
```

Comment:
It may not be necessary to delete the email and name fields when Edit button is checked, as users can change these values in both cases.